docs/static/upgrading.asciidoc

```diff
@@ -43,6 +44,16 @@ This procedure downloads the relevant Logstash binaries directly from Elastic.
 some Logstash plugins have changed in the 5.x release.
 5. Restart your Logstash pipeline after updating your configuration file.
 
+[[upgrading-logstash-pqs]]
+=== Upgrading with Persistent Queues Enabled
+
+Upgrading Logstash with persistent queues enabled is supported. The persistent
+queue directory is self-contained and can be read by a new Logstash instance
+running the same pipeline. You can safely shut down the original Logstash
+instance, spin up a new instance, and set `path.queue` in the `logstash.yml`
+<<logstash-settings-file,settings file>> to point to the original queue directory.
+You can also use a mounted drive to make this workflow easier.
```

Comment:
@suyograo I'm going to merge my change below. I expanded your statement a bit, but I might need to correct the statement if I misunderstood your reason for wanting the clarification.